pkg/ccl/changefeedccl/helpers_test.go

```diff
@@ -1349,7 +1438,9 @@ func makeFeedFactoryWithOptions(
 		f := makeKafkaFeedFactoryWithConnectionCheck(t, srvOrCluster, db, options.forceKafkaV1ConnectionCheck)
 		userDB, cleanup := getInitialDBForEnterpriseFactory(t, s, db, options)
 		f.(*kafkaFeedFactory).configureUserDB(userDB)
-		return f, func() { cleanup() }
+		return f, func() {
```

Comment:
nit: i'm aware this is only a whitespace change but since we're changing the code, maybe we could just have it return `cleanup` directly? ditto for below